### DISCUSSION TITLE
Add a better 'What is RKE2?'

### DISCUSSION THREAD

**Discussion by melroy89** [rancher/rke2#7191] [Ideas]
I believe [the introduction page ](https://docs.rke2.io/)is really lacking a basic "What is RKE2?"..

I would be in favor of creating an additional header with "What is RKE2?". Explaining to people who are new to Kubernetes and just want to know what to expect from RKE2 in term of features and what problem it tries to solve.

The current (2 years old) introduction is not really saying anything about what RKE2 does and can do:

> To meet these goals, RKE2 does the following:
> 
>   - Provides [defaults and configuration options](https://docs.rke2.io/security/hardening_guide) that allow clusters to pass the CIS Kubernetes Benchmark [v1.6](https://docs.rke2.io/security/cis_self_assessment16) or [v1.23](https://docs.rke2.io/security/cis_self_assessment123) with minimal operator intervention
>   - Enables [FIPS 140-2 compliance](https://docs.rke2.io/security/fips_support)
>   - Regularly scans components for CVEs using [trivy](https://github.com/aquasecurity/trivy) in our build pipeline

I think k3s has a better introduction: https://docs.k3s.io/

Thanks.

Kind regards,
Melroy van den Berg

**Comment by brandond** [comment]
This sort of thing is owned by SUSE Product Management to a greater extent than K3s, since K3s is a CNCF project. Glad to point them at this conversation though!

**Reply by melroy89** [reply]
I hope somebody or someone could improve this introduction page
